shared/login/relogin/index.stories.tsx

```diff
@@ -1,7 +1,10 @@
 import * as React from 'react'
 import * as Sb from '../../stories/storybook'
 import Login, {Props} from '.'
+import * as Constants from '../../constants/config'
 
+const makeAccount = (username: string) =>
+  Constants.makeConfiguredAccount({hasStoredSecret: !username.startsWith('a'), username})
```

Comment:
this is a little too tricky (even for a story). lets just make a username `'noSecret'` or something